libs/common-services/src/main/java/uk/gov/di/ipv/core/library/service/IpvSessionService.java

```diff
@@ -120,8 +120,13 @@ public IpvSessionItem generateIpvSession(
         if (errorObject == null) {
             if (isReverification) {
                 ipvSessionItem.pushState(new JourneyState(REVERIFICATION, START_STATE));
+                // Currently reverifcation journeys don't check the user's existing profile so we
+                // have to hard code this to P2 here. Eventually this can be set in the first step
+                // of the reverification journey.
+                ipvSessionItem.setTargetVot(Vot.P2);
```

Comment:
@shivanshuit914 added something to [ClientOAuthSessionDetailsService.java](https://github.com/govuk-one-login/ipv-core-back/pull/2168/files#diff-0384087424c1800866db67985f333eed3e5ab7c662b4f0af4da055d24cea30f7) to set a hardcoded vot when vtr is empty - can this now be removed?

Reply:
Hmm, I don't think so. It does look like it caused by the same root though.

Reply:
I'm checking with Joe, perhaps we can remove Shiv's changes in the follow up PR